scripts/README.md

```diff
@@ -1,13 +1,33 @@
-#Loading data for development
+# Loading data for development
 
-pnp_incident_schema.json requires 'incidents_and_sites.csv'
+## Requirements
 
-pnp_incident_schema_v2.json requires 'public.csv'
+- Python 2.x.
+- python-gdal
 
-incident_schema_v3.json requires the directory `data_for_v3`
+These OS-level requirements can be bootstrapped off of Azavea's GDAL docker
+image ([code](https://github.com/azavea/docker-gdal),
+[Quay repo](https://quay.io/repository/azavea/gdal?tag=latest&tab=tags)).
 
-load_black_spots.py requires `black_spots.json`
+Install Python requirements with pip:
 
-load_interventions.py requires `interventions_sample_pts.geojson`
+```
+pip install -U -r requirements.txt
+```
 
-generate_training_input.py requires `blackspot_training/roads_utm.*` and `blackspot_training/all_crashes_2008-2012.csv`
+## Script and schema dependencies
+
+Certain ETL scripts and schemas in this repo depend on specific data
+sources to run properly. Retrieve these dependencies from the fileshare and
```

Comment:
Outside users won't have access to our fileshare, so I don't think we should include this in the setup instructions. What might be nice is to add some of the files below to a sample_data directory so that they're in the repo. However, the incident data is from the Philippines and I think we should get authorization before including it. Perhaps we could just include these for now:
- `black_spots.json`
- `interventions_sample_pts.geojson`

That will at least allow users to generate a schema with black spots and interventions, and then perhaps manually generate some data using the web UI.

Reply:
Sounds great! I'll create the `sample_data` directory. I think the dependency table is still helpful if you have access to the fileshare, since it's the only documentation that exists for matching scripts and schemas to the example data that they rely on, but I agree that it's weird to have open docs that reference a private data directory. Is it OK to leave the table here, or would it be better to move it to a README in the fileshare folder?

Reply:
I think a README in the fileshare folder would be perfect, that's a great solution!